.github/workflows/create-daily-docs-sync-pr.yaml

```diff
@@ -101,7 +101,7 @@ jobs:
 
           # NOTE: The script will end up not creating a merge commit if there are no new upstream changes.
           # In that case we expect that the PR action will simply not create a PR.
-          ../bot/scripts/pull-and-resolve-english-changes.sh develop "$pr_title"
+          ../bot/scripts/pull-and-resolve-english-changes.sh english/develop "$pr_title"
```

Comment:
Shouldn't this change and the one in the [pull-and-resolve-english-changes.sh](https://github.com/solidity-docs/.github/pull/39/files#diff-b5a9f09a44cf3b1ea862349eefb5fa0315f6524f56221711ab7af91540dbebca) script be included in [#42](https://github.com/solidity-docs/.github/pull/42) instead?

Reply:
No, it's independent of #42. I'm adjusting `pull-and-resolve-english-changes.sh` specifically so that the new script can use it.

`pull-and-resolve-english-changes.sh` used to automatically prepend the remote name, which is fine when we're using a branch - as was always the case until now. The new script needs to use a tag instead.